client/src/components/FullLogo/Logo.css

```diff
@@ -0,0 +1,44 @@
+.logo{
+    display: flex;
+    flex-direction: row;
+    margin: auto;
+}
+
+.modern {
+    font-family: Raleway, sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 44px;
+    letter-spacing: 0.1px;
+    color: #25282B;
+}
+
+.trouseau {
```

Comment:
Not sure if the misspelling was intentional but the company name is spelled with two `s` not one. 
It should be `trousseau` not `trouseau`.